presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcMetadataConfig.java

```diff
@@ -46,4 +49,18 @@ public JdbcMetadataConfig setAllowAggregationPushdown(boolean allowAggregationPu
         this.allowAggregationPushdown = allowAggregationPushdown;
         return this;
     }
+
+    @Min(0)
+    public int getDomainSizeThreshold()
+    {
+        return domainSizeThreshold;
+    }
+
+    @Config("domain-size-threshold")
```

Comment:
`domain-compaction-threshold`   as in `io.prestosql.plugin.hive.HiveConfig#setDomainCompactionThreshold`?